jekyll/_cci2/persist-data.md

```diff
@@ -67,7 +71,7 @@ All data persistence operations within a job will accrue network and storage usa
 * Uploading artifacts
 * Uploading test results
 
-To determine which jobs utilize the above actions, you can search for the following commands in your project's config.yml file:
+To determine which jobs utilize the above actions, you can search for the following commands in your project's `config.yml` file:
```

Comment:
I wonder if we should always use `.circleci/_config.yml` to be totally clear when talking about CircleCI config. I am guessing people might have a quite a few `config.yml`s kicking around depending on what other stuff they are using. What do you think?

Reply:
Yeah that's true, that's a good idea. That probably needs to be its own task as there are many instances throughout the docs that reference simply config.yml